res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -150,7 +149,8 @@ limitations under the License.
         }
 
         .mx_MessageActionBar {
-            right: -100px; // to make sure it doesn't overflow to the left or cover sender profile
+            inset-inline-start: calc(100% - var(--MessageActionBar-size-box));
+            right: initial; // Reset the default value
```

Comment:
This line resets the default value to fix this:

![before](https://user-images.githubusercontent.com/3362943/168134672-44d6bf6b-a0c7-4b53-beac-1e505b5e53f8.png)